docs/source/config_files.rst

```diff
@@ -30,6 +55,14 @@ You can also install files from pip in your ``environment.yml`` as well.
 Our example `environment.yml <https://github.com/binder-examples/python-conda_pip/blob/master/environment.yml>`_
 shows how one can specify a conda environment for repo2docker.
 
+You can also specify which Python version you are using with
+``environment.yml``. By default, ``repo2docker`` will assume you are using
+Python 3.6 with your ``environment.yml`` unless you include the version of
+Python in the file.  ``conda`` supports Python versions 2.7, 3.4, 3.5, and 3.6.
```

Comment:
I would reverse the Python versions from highest to lowest (3.6, 3.5...) to deemphasize Python 2.